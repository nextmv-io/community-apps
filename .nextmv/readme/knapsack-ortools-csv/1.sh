python3 main.py -duration 30
