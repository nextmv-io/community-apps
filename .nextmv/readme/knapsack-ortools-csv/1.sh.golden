{
  "statistics": {
    "result": {
      "custom": {
        "constraints": 1,
        "provider": "SCIP",
        "status": "optimal",
        "variables": 11
      },
      "duration": 0.123,
      "value": 444.0
    },
    "run": {
      "duration": 0.123
    },
    "schema": "v1"
  }
}
